examples/oidc/docker-compose.yml

```diff
@@ -21,6 +25,9 @@ services:
       - "587:587"
       - "993:993"
       - "8000:8000"
+    environment:
+      - OIDC_CONFIGURATION_URL=https://keycloak:8080/auth/realms/oidc/.well-known/openid-configuration
```

Comment:
`https`? I don't think it supports HTTPS

Reply:
what does not support https? the local keycloak or the james imap and smtp oauth protocol?

Reply:
just a keycloack url 
`https://keycloak:8080...`